qa/sc_bwt_minimum_value.py

```diff
@@ -39,31 +42,36 @@
           and then to MC/SC blocks.
         - verify epoch 0 certificate, verify backward transfers list
 """
-class SCBwtMinValue(SidechainTestFramework):
 
+
+class SCBwtMinValue(SidechainTestFramework):
     sc_nodes_bootstrap_info = None
     sc_withdrawal_epoch_length = 10
 
     def setup_nodes(self):
         num_nodes = 1
         # Set MC scproofqueuesize to 0 to avoid BatchVerifier processing delays
-        return start_nodes(num_nodes, self.options.tmpdir, extra_args=[['-debug=sc', '-logtimemicros=1', '-scproofqueuesize=0']] * num_nodes)
+        return start_nodes(num_nodes, self.options.tmpdir,
+                           extra_args=[['-debug=sc', '-logtimemicros=1', '-scproofqueuesize=0']] * num_nodes)
 
     def sc_setup_chain(self):
         sc_creation_zens = 100
```

Comment:
Done

Reply:
@sgonta check this comment, we need to use `sc_creation_zens` value instead of hardcoded `100`
https://github.com/HorizenOfficial/Sidechains-SDK/pull/574/files#diff-7fcbb86c173eb249df6cb41115de3b9df0218da3de6366f50db7d98fa62aa84bR69

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/110401079/205672917-2b55f30e-5f2f-4096-a574-c21362d6a608.png)